SourceCode/raytracer/hw4.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+void hw4()
+{
+    std::cout << "Task 2" << std::endl;
```

Comment:
You could "using namespace std" in the function scope, preventing some of the boiler code.